packages/core/src/lib/androidSdk/AndroidSdkTools.ts

```diff
@@ -141,11 +141,11 @@ export class AndroidSdkTools {
     const env = this.getEnv();
     const zipalignCmd = [
       `"${this.pathJoin(this.getAndroidHome(), `/build-tools/${BUILD_TOOLS_VERSION}/zipalign`)}"`,
-      '-v -f -p 4',
+      '-c -v -p 4',
```

Comment:
Could you add some documentation for what these flags are and why we need them?

Reply:
per documentation, `-c` checks of a file was zipaligned correctly. I don't think that's what we want here. The `-f` flag we had there was to overwrite the output file, if it already existed.

If we need a command to check alignment, I'd rather have a separate function so we keep backward compatibility with other tools using the core library (eg: PWA Builder)

Reply:
Am I understanding correctly that, when using `com.github.triplet.play` Gradle will zipalign the file?

Reply:
So `com.github.triplet.play` will not zipalign the file, but the updates to the gradle version (which `com.github.triplet.play` requires) does automatically zipalign the file. The `-f` flag I believe is no longer required at this point and instead a `fs.copyFile ` change actually does the part that overwrites and copies the file after validation.

Reply:
@PEConn (https://developer.android.com/studio/command-line/zipalign) Did you want me to copy the flag definitions into the file as a comment?

Reply:
Verbose is probably self explanatory - explaining why we need `c` would probably be helpful though.

Reply:
@nohe427 I don't see a behaviour change regarding zipalign between versions 4.0.0 and 4.1.0 of the gradle plugin (https://developer.android.com/studio/releases/gradle-plugin#behavior_changes). I think this behaviour has been there [since 2.2.0](https://developer.android.com/studio/releases/gradle-plugin#2-2-0)? Is this changed needed for this PR to work?

If that's the case, the old zipalign call doesn't break anything, but also doesn't have any effect and should remove the `zipalign` call from the `build` command. If we want to double check if the file is properly aligned, let's add another function that verifies the alignment.

Reply:
@PEConn - I added info about all the flags (no flag left behind!) + a documentation link. My reasoning is that it might be worthwhile for folks unfamiliar with zipalign to see the docs themselves without searching too much. LMK if I should trim it down.

Reply:
@andreban - I added a command called zipalignOnlyVerification. I think this is what you were interested in having instead?